### ISSUE TITLE
Add support for names in m3u playlist

### ISSUE THREAD

**Issue by ValdikSS** [xylosper/bomi#7]
Hello!
If you have an .m3u playlist with names, e.g. for IPTV, you expect names to be shown, not URL.
# EXTINF:-1,MTV Dance

http://...
# EXTINF:-1,MTV Hits UK

http://...
# EXTINF:-1,MTV Rocks

http://...

Should be shown:
MTV Dance
MTV Hits UK
MTV Rocks

Actual result:
http://...
http://...
http://...

**Comment by ValdikSS** [comment]
Doesn't work with playlist in the following format:

#EXTINF:-1 group-title="Региональные" tvg-name="TV_Bakhoristan" tvg-logo="",TV Bakhoristan
http://127.0.0.1:8000/pid/b10151ed052d5e501e653c8bfa7c51dc60312a11/stream.mp4
#EXTINF:-1 group-title="Региональные" tvg-name="TV_Safina" tvg-logo="",TV Safina
http://127.0.0.1:8000/pid/d382c123007f1bbd0ccc6d2acc6767c250b1f470/stream.mp4
#EXTINF:-1 group-title="Развлекательные" tvg-name="TV_Sale" tvg-logo="",TV Sale
http://127.0.0.1:8000/pid/55a2a753222a82f5e170b955b35f46d857888c1c/stream.mp4

**Comment by xylosper** [comment]
Because it's illformed. The timelength information(in this case, unknown(-1)) should be followed by a comma(,) like:
```
#EXTINF:-1, group-title="Региональные" tvg-name="TV_Bakhoristan" tvg-logo="",TV Bakhoristan
```

**Comment by ValdikSS** [comment]
I'm afraid it's not malformed. There shouldn't be comma.

**Comment by xylosper** [comment]
Please see: http://tools.ietf.org/html/draft-pantos-http-live-streaming-14#section-4.3.2.1

**Comment by ValdikSS** [comment]
This format introduced by Borpas in his IPTV Player long ago before this draft.
http://borpas.info/iptvplayer-docs
http://xmtvplayer.com/build-m3u-file

**Comment by xylosper** [comment]
1. Specification is not about who did first. It's not a patent.
2. It is denoted as **extended format** in Borpas, too, unless the translator was wrong:
```
It is recommended to use the extended format of the channel list with the settings
```

Anyway, it's implemented because this can be done without breaking standard parser :)
Also, I found that the name in playlist wasn't displayed at all including other standard m3u, so I fixed it too.
